README.md

```diff
@@ -1,31 +1,33 @@
 # HELICS Matlab interface
+
 This repository contains the tools and files required for using HELICS with Matlab.
 For HELICS documentation see [HELICS.org](www.helics.org)
 
 ## Package Download
 
 For systems running standard newer windows or mac(non arm based) and a newer Matlab (2021a+) there is a zip or tar.gz file
-with the entire system that can be downloaded and unziped. The zip file contains a helics folder that should be added to the matlab path(not with subfolders).
-These archives can be found attached to the releases. 
+with the entire system that can be downloaded and unzipped. The zip file contains a helics folder that should be added to the matlab path(not with subfolders).
+These archives can be found attached to the releases.
 There is also a release that should work with ubuntu 20.04,  It is unclear if this mex file will work on other linux platforms.
 
 ## Installation and Requirements
-For other platforms the installation is as simple as downloading the repository and inside matlab executing the buildHelicsInterface Matlab script.
 
-This should download the requried HELICS binary files and compile the platform specific mex file.  This does require a compiler be installed on the system and executing
+For other platforms the installation is as simple as downloading this repository and inside matlab executing the `buildHelicsInterface` Matlab script.
+
+This should download the requried HELICS binary files and compile the platform specific mex file.  This does require a compiler be installed on the system and executing inside matlab
 
 ``` matlab
 >> mex -setup
 ```
 
-inside matlab prior to running the script.  
+This setup is only once on any given Matlab installation and does not need to be repeated is upgrading HELICS 
 
 ```matlab
 >> cd matHELICS
 >> buildHelicsInterface('../helics');
 ```
 
-this will generate the helics interface in a folder 1 level above the matHELICS directory.  If desired it can also be build in the matHELICS directory 
+this will generate the helics interface in a folder 1 level above the matHELICS directory.  If desired it can also be build in the matHELICS directory for example:
```

Comment:
```suggestion
this will generate the helics interface in a folder 1 level above the matHELICS directory.  If desired it can also be built in the matHELICS directory for example:
```